datafusion/expr/src/expr_fn.rs

```diff
@@ -366,8 +366,8 @@ unary_scalar_expr!(Round, round, "round to nearest integer");
 unary_scalar_expr!(Trunc, trunc, "truncate toward zero");
 unary_scalar_expr!(Abs, abs, "absolute value");
 unary_scalar_expr!(Signum, signum, "sign of the argument (-1, 0, +1) ");
-unary_scalar_expr!(Exp, exp, "base 2 logarithm");
-unary_scalar_expr!(Log2, log2, "base 10 logarithm");
+unary_scalar_expr!(Exp, exp, "exponent");
```

Comment:
Very minor thing (mainly for consistency), but since most of the functions here mirror the [Postgres documentation](https://www.postgresql.org/docs/7.1/functions-math.html), maybe we should use it for the `exp()` as well.

```suggestion
unary_scalar_expr!(Exp, exp, "exponential");
```